### ISSUE TITLE
[BUG] java.lang.IllegalStateException: Tile is null, asynchronous access

### ISSUE THREAD

**Issue by berkayakin** [bloodmc/GriefDefender#445]
- **Versions**
Spigot/Paper/Sponge: Purpur 1.18.2
GriefDefender: 2.1.3
LuckPerms: 5.4.9

- **Describe the bug**
The plugin is spamming the console with the same warning for different blocks.

```
[13:22:53 WARN]: Exception in thread "pool-17-thread-26665" java.lang.IllegalStateException: Tile is null, asynchronous access? CraftBlock{pos=BlockPosition{x=-1504, y=91, z=-1156},type=CHEST,data=Block{minecraft:chest}[facing=west,type=right,waterlogged=false],fluid=net.minecraft.world.level.material.FluidTypeEmpty@288b2742}
[13:22:53 WARN]:        at com.google.common.base.Preconditions.checkState(Preconditions.java:590)
[13:22:53 WARN]:        at org.bukkit.craftbukkit.v1_18_R2.block.CraftBlockStates$BlockEntityStateFactory.createBlockState(CraftBlockStates.java:89)
[13:22:53 WARN]:        at org.bukkit.craftbukkit.v1_18_R2.block.CraftBlockStates$BlockEntityStateFactory.createBlockState(CraftBlockStates.java:75)
[13:22:53 WARN]:        at org.bukkit.craftbukkit.v1_18_R2.block.CraftBlockStates.getBlockState(CraftBlockStates.java:263)
[13:22:53 WARN]:        at org.bukkit.craftbukkit.v1_18_R2.block.CraftBlockStates.getBlockState(CraftBlockStates.java:221)
[13:22:53 WARN]:        at org.bukkit.craftbukkit.v1_18_R2.block.CraftBlockStates.getBlockState(CraftBlockStates.java:206)
[13:22:53 WARN]:        at org.bukkit.craftbukkit.v1_18_R2.block.CraftBlock.getState(CraftBlock.java:322)
[13:22:53 WARN]:        at com.griefdefender.util.SignUtil.isSign(SignUtil.java:122)
[13:22:53 WARN]:        at com.griefdefender.listener.BlockEventHandler.trackBlockPhysics(BlockEventHandler.java:623)
[13:22:53 WARN]:        at com.griefdefender.listener.BlockEventHandler.lambda$onBlockNotify$0(BlockEventHandler.java:601)
[13:22:53 WARN]:        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
[13:22:53 WARN]:        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[13:22:53 WARN]:        at java.base/java.lang.Thread.run(Thread.java:833)
```

- **Steps to reproduce the behavior**
Usually placing beds or liquids causes that. Mostly with iron farms etc.

- **Expected behavior**
The console to be spammed by GriefDefender.

**Comment by berkayakin** [comment]
It is possible the plugin uses the API method async and it is incorrect.

**Comment by bloodmc** [comment]
Fixed in GD 2.1.5
